.pylintrc

```diff
@@ -38,7 +38,7 @@ load-plugins=linter_plugin
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=fixme,locally-disabled,abstract-class-not-used
+disable=fixme,locally-disabled,abstract-class-not-used,bad-continuation,too-few-public-methods,no-self-use
```

Comment:
ah, it's `bad-continuation`... that's why EOL backslash wasn't caught at https://github.com/letsencrypt/letsencrypt/pull/733/files#r38259734; this is really not something that should be disabled